js/helper.js

```diff
@@ -13,6 +15,23 @@ const BINARY_DISTRIBUTIONS = [
   { packageName: '@sentry/cli-win32-i686', subpath: 'bin/sentry-cli.exe' },
 ];
 
+/**
+ * This convoluted function resolves the path to the manually downloaded fallback
+ * `sentry-cli` binary in a way that can't be analysed by @vercel/nft.
+ *
+ * Without this, the binary can be detected as an asset and included by bundlers
+ * that use @vercel/nft.
+ *
+ * @returns {string} The path to the sentry-cli binary
+ */
+function getFallbackBinaryPath() {
+  const parts = [];
+  parts.push(__dirname);
+  parts.push('..');
+  parts.push(`sentry-cli${process.platform === 'win32' ? '.exe' : ''}`);
+  return path.resolve(...parts);
```

Comment:
```suggestion
  return path.resolve(__dirname, '..', `sentry-cli${process.platform === 'win32' ? '.exe' : ''}`);
```

Reply:
As per the comment above this function is convoluted like this so that static code analyzers (like [Vercel's nft](https://github.com/vercel/nft), and pretty much only Vercel's nft) cannot detect the binary as a "dependency" (dependency here meaning a thing that needs to be included in an AWS lambda for the application to work), because generally speaking Sentry CLI is not a runtime dependency but one that is only required during builds.